app/templates/views/guidance/features/who-can-use-notify.html

```diff
@@ -36,10 +36,10 @@ <h2 class="govuk-heading-m" id="gp">GP surgeries</h2>
     NHS-funded GP surgeries can use GOV.UK&nbsp;Notify.
   </p>
   <p class="govuk-body">
-    From 1 April 2024, GP surgeries:
+    NHS-funded GP surgeries can use GOV.UK Notify, but they:
   </p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>will not get an <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_pricing_text_messages') }}">annual allowance of free text messages</a></li>
+    <li>do not get an annual allowance of free text messages <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_pricing_text_messages') }}">annual allowance of free text messages</a></li>
```

Comment:
Remove duplication and keep link

```suggestion
    <li>do not get an <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_pricing_text_messages') }}">annual allowance of free text messages</a></li>
```